protocol/communities/community.go

```diff
@@ -898,6 +901,27 @@ func (o *Community) IsMemberInChat(pk *ecdsa.PublicKey, chatID string) bool {
 	return o.getChatMember(pk, chatID) != nil
 }
 
+// Uses bloom filter members list to estimate presence in the channel.
+// False positive rate is 0.1%.
+func (o *Community) IsMemberInChatsBloomFilter(chatID string) bool {
+	if o.IsControlNode() || !o.channelEncrypted(chatID) {
```

Comment:
In theory, all admins would also be in all chats. So maybe we can add them to this condition?

Reply:
Admins can miss encryption keys as well, so they need to estimate it from bloom filter.

Reply:
Ah, you are right :bulb: